cl/corpus_importer/management/commands/recap_into_opinions.py

```diff
@@ -89,6 +49,8 @@ def import_opinions_from_recap(
 
         # Manually select the replica db which has an addt'l index added to
         # improve this query.
```

Comment:
This is a one-time script, right? Let's make sure we remove that index when we're done. Can we make a sub-issue to remember to do that? We need to be really careful not to permanently add indexes we won't use in the future.

Reply:
Well.. it was a one time script until it wasn't right?  Regardless yes we should.

Reply:
Great. Can we make that issue or sub-issue, please?

Reply:
√